app/Commands/ReposCommand.php

```
@@ -0,0 +1,413 @@
+                case 'Windows':
+                    shell_exec("start '{$url}' > /dev/null 2>&1");
+                    break;
+                case 'Linux':
+                    shell_exec("xdg-open '{$url}' > /dev/null 2>&1");
+                    break;
+            }
+            info('🌐 Opened in browser');
+        } catch (\Exception $e) {
+            error('Failed to open browser');
+        }
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'openInBrowser'. (Unused Code Rules)

(UnusedPrivateMethod)